mqtt-cs/src/main/java/org/apache/rocketmq/mqtt/cs/session/loop/SessionLoopImpl.java

```diff
@@ -275,11 +275,11 @@ private void addSubscriptionAndInit(Session session, Set<Subscription> subscript
             return;
         }
         session.addSubscription(subscriptions);
-        AtomicInteger result = new AtomicInteger(subscriptions.size());
         for (Subscription subscription : subscriptions) {
             queueFresh.freshQueue(session, subscription);
             Map<Queue, QueueOffset> queueOffsets = session.getQueueOffset(subscription);
             if (queueOffsets != null) {
+                AtomicInteger result = new AtomicInteger(queueOffsets.size());
                 for (Map.Entry<Queue, QueueOffset> entry : queueOffsets.entrySet()) {
```

Comment:
Do we really need the 'result' variable? 
we just need to judge whether the future is complete or not, and only complete once?